src/libostree/ostree-diff.h

```diff
@@ -69,7 +69,9 @@ gboolean ostree_diff_dirs (OstreeDiffFlags flags,
                            GPtrArray      *removed,
                            GPtrArray      *added,
                            GCancellable   *cancellable,
-                           GError        **error);
+                           GError        **error,
```

Comment:
Unfortunately, this is public API, so we can't break it now.  We'd need to add a new version, like `ostree_diff_dirs_with_options()` or something.  Also, the `GError` parameter should be last.